Library/ViewModels/ProjectPamphletContentViewModelTests.swift

```diff
@@ -471,6 +476,84 @@ final class ProjectPamphletContentViewModelTests: TestCase {
     }
   }
 
+  func testLoadProjectIntoDataSource_CreatorDetailsLoaded_ControlVariant() {
+    let project = Project.template
+
+    let envelope = ProjectCreatorDetailsEnvelope(backingsCount: 25, id: "123", launchedProjectsCount: 50)
+
+    let mockService = MockService(fetchProjectCreatorDetailsResult: .success(envelope))
+
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~ [
+        OptimizelyExperiment.Key.nativeProjectPageConversionCreatorDetails.rawValue: OptimizelyExperiment.Variant.control.rawValue
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/8676c26a01ecffa0f982d0e5b09f2191cd795621/Library/ViewModels/ProjectPamphletContentViewModelTests.swift#L488"></span>Line should be 110 characters or less: currently 130 characters<br />
<code>line_length</code> <code>ProjectPamphletContentViewModelTests.swift:488</code></td>
    </tr>
  </tbody>
</table>